h5py/tests/old/test_h5d_direct_chunk_write.py

```diff
@@ -32,3 +32,56 @@ def test_write_direct_chunk(self):
         for i in range(10):
             read_data = filehandle["data"][i]
             self.assertTrue((array[i] == read_data).all())
+
+
+@ut.skipUnless(h5py.version.hdf5_version_tuple >= (1, 10, 2), 'Direct Chunk Reading requires HDF5 >= 1.10.2')
+class TestReadDirectChunk(TestCase):
+    def test_read_offsets(self):
+
+        filename = self.mktemp().encode()
+        filehandle = h5py.File(filename, "w")
+
+        slice = numpy.arange(16).reshape(4, 4)
```

Comment:
Can we call this variable something else? `slice` is a Python builtin, so it confused me for a moment looking at the code.

Reply:
Oups, yes, thanks for the feedback. I fixed that on the following commits.